unreleased_history/public_api_changes/add_new_compaction_parameter.md

```diff
@@ -0,0 +1 @@
+Expose delete ratio parameter from the compaction API to the C API
```

Comment:
Could you be more specific here, something like "Add parameter `deletion_ratio` to C API  `rocksdb_options_add_compact_on_deletion_collector_factory`"?

Reply:
thanks for the restart, I updated the release note. let me know if anything is missing.